android/src/main/java/io/wazo/callkeep/RNCallKeepModule.java

```diff
@@ -491,6 +493,76 @@ public void toggleAudioRouteSpeaker(String uuid, boolean routeSpeaker) {
         }
     }
 
+    @ReactMethod
+    public void setAudioRoute(String uuid, String audioRoute, Promise promise){
+        try {
+            VoiceConnection conn = (VoiceConnection) VoiceConnectionService.getConnection(uuid);
+            if (conn == null) {
+                return;
+            }
+            if(audioRoute.equals("Bluetooth")) {
+                conn.setAudioRoute(CallAudioState.ROUTE_BLUETOOTH);
```

Comment:
Can you add logs (eg: `Log.d(Tag, ...)`) about the action and the current audioRoute at this point ?

Reply:
Done!